risc0/cargo-risczero/src/lib.rs

```diff
@@ -48,6 +48,7 @@ pub struct Risczero {
     pub command: RisczeroCmd,
 }
 
+#[non_exhaustive]
```

Comment:
I'm making this non-exhaustive so future changes are not considered major breaking changes in the semver check.

Reply:
This doesn't make sense for `cargo-risczero`, it doesn't need to maintain backwards compabilitity (at the API-level, only at the command-line level).